ansible/06_osp_storage.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: localhost
+  gather_facts: false
+  vars_files: vars/default.yaml
+  tasks:
+    - block:
+        - name: Provision and install OCS storage
+          include: ocs.yaml
+      when: ocs_enabled == true and use_kni_ipi_virt == true
```

Comment:
is use_kni_ipi_virt requried? I don't its being set anywhere.

Reply:
Ah, I missed removing that check.  Thank you!